src/test/checktest.sh

```diff
@@ -28,15 +28,16 @@ EOF
    ;;
 esac
 
-RESET=`shopt -p nullglob`  # Save current nullglob setting
-shopt -s nullglob          # Return an empty string for failed wildcard matches 
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+RESET=`shopt -p nullglob`                # Save current nullglob setting
+shopt -s nullglob                        # Return an empty string for failed wildcard matches
 if [ "x$1" = "x" ]
 then
-   testdirs=`echo TEST*`   # All TEST directories
+   testdirs=`echo ${SCRIPT_DIR}/TEST*`   # All TEST directories
```

Comment:
Why did you make this change?  The output from checktest.sh is not as clean as a result.

Reply:
Hi Rob, this change enables calling `checktest.sh` from other folders. This is needed by the `make check` and `make checkpar` targets in the cmake build. Do you have a comparison of the output before and after easily available? I wanted to understand why the output is not as clean as before

Reply:
Ok, I see now, thanks! Fixed in [7a45de3](https://github.com/hypre-space/hypre/pull/1181/commits/7a45de34104f4efce7663051a7b272c59ebffa00)

Reply:
I understand that you want to be able to run `<path-to-checktest.sh>/checktest.sh` but does the output really need to show `<full-path-to-checktest.sh>` regardless of what `<path-to-checktest.sh>/` is?  I think the simple output is enough since the tests are required to be in the same directory as `checktest.sh` so the path to that script must be known.  Also, by far our most common use of the script will be to use it within its directory.

Reply:
Looks good.  Thanks!

Reply:
Thanks, Rob! Yes, I agree that reporting the relative path is enough